### ISSUE TITLE
Conformance test fails as iptables is not installed by default in rhel8

### ISSUE THREAD

**Issue by ShylajaDevadiga** [rancher/rke2#356]
RKE2 Version: rke2 version v1.18.8-beta20+rke2
OSL RHEL 8

Issue:
Create a two node cluster one master one worker
Install sonobuoy

Issue is not seen in RHEL 7.8, as iptables is available

Check the logs:
```
  Normal   Scheduled               <unknown>            default-scheduler                                    Successfully assigned projected-9168/pod-projected-configmaps-b6db5404-7173-4b8b-b0e6-db64b98d28c9 to ip-172-31-46-21.us-east-2.compute.internal
  Warning  FailedCreatePodSandBox  4m21s                kubelet, ip-172-31-46-21.us-east-2.compute.internal  Failed to create pod sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox "02ea318fefdbbd590f765f9d7125f82c2b66f3ee732a79dc5cb12bf3b621884f": failed to allocate for range 0: no IP addresses available in range set: 10.42.1.1-10.42.1.254
  Warning  FailedCreatePodSandBox  4m7s                 kubelet, ip-172-31-46-21.us-east-2.compute.internal  Failed to create pod sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox "f133bd2a5f3e1f6da02d04ffde888e65fe962d27ff0e5a3f5719735fc7e3a12b": failed to allocate for range 0: no IP addresses available in range set: 10.42.1.1-10.42.1.254
```

**Comment by dweomer** [comment]
Determined to be missing iptables on el8 installation. https://github.com/rancher/rke2-packaging/pull/4 should address

**Comment by ShylajaDevadiga** [comment]
Validated, iptables is available in RKE2 Version: rke2 version v1.18.8-beta20+rke2 with updated rpms
Conformance test passed as well on a 1 master 1 agent cluster
```
sonobuoy results $res
Plugin: e2e
Status: passed
Total: 4992
Passed: 277
Failed: 0
Skipped: 4715

Plugin: systemd-logs
Status: passed
Total: 2
Passed: 2
Failed: 0
Skipped: 0 
```
